rfc6962-bis.xml

```diff
@@ -359,7 +359,7 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
       <section title="Redaction of Domain Name Labels" anchor="domain_name_redaction">
         <section title="Redacting Labels in Precertificates" anchor="redacting_labels">
           <t>
-            When creating a precertificate, the CA MAY omit the subjectAltName extension. Instead, the CA MUST include a <xref target="redacted_san_extension">redactedSubjectAltName</xref> extension that contains, in a redacted form, the same identities that will be included in the certificate's subjectAltName extension.
+            When creating a precertificate, the CA MAY omit the the subjectAltName extension, even if it intends to include the extension in the final certificate. Instead, the CA MUST include a <xref target="redacted_san_extension">redactedSubjectAltName</xref> extension that contains, in a redacted form, the same identities that will be included in the certificate's subjectAltName extension.
```

Comment:
Why the added "the" ?

BTW, the use of "Instead" seems a bit vague.
How about: s/Instead,/If omitting the subjectAltName extension,/g